bin/cli.js

```diff
@@ -9,7 +9,7 @@ if(process.argv.length < 3 || !/^https?:\/\//.test(process.argv[2])){
 	return;
 }
 
-require("./getURL.js")(process.argv[2], process.argv[3] === "html" ? "html" : "text", function(result){
+require("./../lib/getURL.js")(process.argv[2], process.argv[3] === "html" ? "html" : "text", function(result){
```

Comment:
Is it okay for it to be in this commit, or you still want it to be a separate commit ?